eth/consensus/clique/__init__.py

```diff
@@ -3,10 +3,7 @@
     CliqueConsensus,
     CliqueConsensusContext,
 )
-from .constants import (  # noqa: F401
-    NONCE_AUTH,
-    NONCE_DROP
-)
+from .constants import NONCE_AUTH, NONCE_DROP  # noqa: F401
```

Comment:
nit: I believe if you add a hanging comma, this will be isorted like: 
```suggestion
from .constants import (   # noqa: F401
    NONCE_AUTH, 
    NONCE_DROP,
)
```
to match the others

Reply:
noop - we have the `skip=__init__.py` set in `tox.ini`. But ran it specifically here to make it prettier :)